feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.service.ts

```diff
@@ -15,6 +15,37 @@ export class ConfiguratorAttributeQuantityService {
     return !value || value === '0';
   }
 
+  /**
+   * Checks if the interaction with the quantity control needs for multiselection components
+   * to be disabled
+   * @param attribute Configurator Attribute
```

Comment:
According to JSDoc we could add a type of the corresponding parameter, for instance: 
`param {Configurator.Attribute} attribute - Configurator attribute`

Reply:
done